server/src/main/java/org/apache/druid/catalog/model/table/BaseInputSourceDefn.java

```diff
@@ -238,6 +241,7 @@ protected InputSource convertSource(
       return jsonMapper.convertValue(jsonMap, inputSourceClass());
     }
     catch (Exception e) {
+      LOG.debug(e, "Invalid input source specification");
```

Comment:
Should this be error log? debug isnt logged by default.

Reply:
This is for debugging. Specifically, the case in which an input format requires a JSON or Guice thing which is not configured in the Coordinator. Only going to happen for developers, not for users.